extensions/amp-ad/0.1/amp-ad.js

```diff
@@ -280,6 +280,10 @@ class AmpAd extends AMP.BaseElement {
             this.updateSize_(newHeight, newWidth);
           }
         }, /* opt_is3P */ true);
+        // Race condition, noContentHandler_() gets called first.
+        if (!this.iframe_) {
```

Comment:
This one can't be the style issue, the single JS thread guarantees there is an `#iframe` present. Are you trying to guard the `render-start` event's `iframe.style` access?